strapi/strapi
# Change strapi icons color in dashboard

### What does it do?

Change the Strapi svg colors in the Dashboard

### Why is it needed?

Because we have a bug in the dashboard, the icons for the blog and careers links seem to be broken.

### How to test it?

Run your Strapi application and open the admin panel. after logging in and navigating to your dashboard, you should see the issue.
![Schermata 2022-11-30 alle 16 00 05](https://user-images.githubusercontent.com/25641936/204347715-7921fa1f-3803-43bb-8a8e-ff32770f9c0e.png)
![Schermata 2022-11-30 alle 15 59 59](https://user-images.githubusercontent.com/25641936/204347727-0924b43c-26d7-479d-9359-eea2a4f0f8b2.png)

now resolved
![Schermata 2022-11-30 alle 16 00 05](https://user-images.githubusercontent.com/2589748/204858323-42871ecb-fa07-4491-8841-b9c6935d7ec2.png)
![Schermata 2022-11-30 alle 15 59 59](https://user-images.githubusercontent.com/2589748/204858326-8ae1795d-9a33-4e55-b832-ef4d582427e8.png)

### Related issue(s)/PR(s)

Fix [#793](https://github.com/strapi/design-system/issues/793)
